src/components/utils/Svg.css

```diff
@@ -6,40 +11,40 @@
 .worker,
 .refresh,
 .add-button {
-  fill: var(--theme-splitter-color);
-}
-
-.worker,
-.folder {
-  position: relative;
-  top: 2px;
+    fill: var(--theme-splitter-color);
 }
 
 .domain,
 .file,
-.worker,
-.refresh,
-.add-button {
-  position: relative;
-  top: 1px;
+.worker {
+    position: relative;
+    top: 1px;
 }
 
 .domain svg,
 .folder svg,
 .worker svg,
 .refresh svg,
 .add-button svg {
-  width: 15px;
+    width: 15px;
 }
 
 .file svg {
-  width: 13px;
+    width: 13px;
 }
 
-.file svg,
-.domain svg,
-.folder svg,
-.refresh svg,
-.worker svg {
-  margin-inline-end: 5px;
+html:not([dir="rtl"]) .file svg,
+html:not([dir="rtl"]) .domain svg,
+html:not([dir="rtl"]) .folder svg,
+html:not([dir="rtl"]) .refresh svg,
+html:not([dir="rtl"]) .worker svg {
+    margin-right: 5px;
+}
+
+html[dir="rtl"] .file svg,
+html[dir="rtl"] .domain svg,
+html[dir="rtl"] .folder svg,
+html[dir="rtl"] .worker svg,
+html[dir="rtl"] .refresh svg {
+    margin-left: 5px;
```

Comment:
@jasonLaster I did the rebase and I tested dark theme, white theme, Chrome, Firefox, tooltip. I did not test rtl since I do not know how but I included the changes. Please accept this pull request since I keep having conflicts with parts of my patch that landed in upstream. Any issue that you find can be fixed in another branch without these merge conflicts.